lexos/interfaces/bubble_viz_interface.py

```diff
@@ -18,11 +18,11 @@
 # Tells Flask to load this function when someone is at '/viz'
 @viz_view.route("/viz", methods=["GET", "POST"])
 def viz():
-    """
-    Handles the functionality on the alternate bubbleViz page with performance
-    improvements.
-    Note: Returns a response object (often a render_template call) to flask and
-     eventually to the browser.
+    """ handles the functionality on the alternate bubbleViz page
```

Comment:
no space between quotes and the one line summary.

Please see how others does the documentation.